src/Components/Helper/ComponentHelper.php

```diff
@@ -14,6 +14,7 @@ class ComponentHelper
 
     const ACTION_ID_PATTERN = "/^action\.[A-z]*\.[A-z_]*$/";
     const INTERPRETER_ID_PATTERN = "/^interpreter\.[A-z]*\.[A-z_]*$/";
+    const NLU_ID_PATTERN = "/^language_service\core*\.[A-z_]*$/";
```

Comment:
Let's update this constant name to be as clear as possible.

```suggestion
    const LANGUAGE_SERVICE_ID_PATTERN = "/^language_service\core*\.[A-z_]*$/";
```